### ISSUE TITLE
dbt should compile an analytics folder to target as well as the models folder

### ISSUE THREAD

**Issue by jthandy** [analyst-collective/dbt#43]
Right now it's super annoying to interpolate the schema names in analytical queries in a dbt project. We've never really considered dbt a tool for analysis, but the way I'm using it is that in a single dbt project you have both models and analyses, and you work on them in parallel in your text editor and sql terminal. Once the results of a model are good, you `dbt run` it. Once the results of an analysis are good, you paste it into Mode. 

The problem is that right now, analyses don't get compiled, which means that schema interpolation is super-annoying. Every analytic query needs to have its schema name changed when moving from test to production (which is exactly what I'm doing right now with 12 queries). 

(Test vs. production is managed via schema names.)

I don't think this is really requesting any new functionality, just the addition of an `analytics` folder within `dbt-project.yml` and to also create a folder for that within `target` that compiles in exactly the same way that `models` currently does.

**Comment by drewbanin** [comment]
"I don't think this is really requesting any new functionality" .... *creates github issue*

good idea, this will be a part of the 0.3.0 release

**Comment by cmerrick** [comment]
`sources` is already allowed to take on multiple values, why not just add your `analysis` folder to the list of sources?

**Comment by drewbanin** [comment]
Yeah, that's what I was thinking. Problem is that analysis code shouldn't get built through dbt. We just want to compile for copy/paste into something like Mode

**Comment by cmerrick** [comment]
is the issue that it will tack a `create view` or `create table` around the query, which we don't want in this case?

**Comment by drewbanin** [comment]
Yeah! That's exactly it
